pkg/forwarder/telemetry.go

```diff
@@ -30,12 +30,13 @@ var (
 	hostMetadataEndpoint  = transaction.Endpoint{Route: "/api/v2/host_metadata", Name: "host_metadata_v2"}
 	metadataEndpoint      = transaction.Endpoint{Route: "/api/v2/metadata", Name: "metadata_v2"}
 
-	processesEndpoint    = transaction.Endpoint{Route: "/api/v1/collector", Name: "process"}
-	rtProcessesEndpoint  = transaction.Endpoint{Route: "/api/v1/collector", Name: "rtprocess"}
-	containerEndpoint    = transaction.Endpoint{Route: "/api/v1/container", Name: "container"}
-	rtContainerEndpoint  = transaction.Endpoint{Route: "/api/v1/container", Name: "rtcontainer"}
-	connectionsEndpoint  = transaction.Endpoint{Route: "/api/v1/collector", Name: "connections"}
-	orchestratorEndpoint = transaction.Endpoint{Route: "/api/v1/orchestrator", Name: "orchestrator"}
+	processesEndpoint        = transaction.Endpoint{Route: "/api/v1/collector", Name: "process"}
+	processDiscoveryEndpoint = transaction.Endpoint{Route: "/api/v1/collector", Name: "processDiscovery"}
```

Comment:
nit: Given the other endpoint names, what do you think of naming it `process_discovery` instead of `processDiscovery` ?

Reply:
Let me give it a shot, I'm not sure if there needs to be changes made in the intake side first

Reply:
[it's just a name here, it should only affect telemetry metrics](https://github.com/DataDog/datadog-agent/blob/main/pkg/forwarder/transaction/endpoint.go#L13)

Reply:
Oh awesome! Yeah I was just about to comment that it didn't seem to make a difference

Reply:
This won't be a problem for the `process_discovery` check but for `process`, `rtprocess`, we're using `-1` to distinguish between a real `0` value and a not collected one:

https://github.com/DataDog/datadog-agent/blob/be59410d2c6d6321289287d8563e2c17493f369f/pkg/process/checks/process.go#L368-L385